Makefile

```diff
@@ -28,7 +28,7 @@ publisher_v2_validation_records := $(publisher_v2_examples:formats/%.json=dist/f
 publisher_v2_links_validation_records := $(publisher_v2_links_examples:formats/%.json=dist/formats/%.json.publisher_v2_links.valid)
 
 # The various scripts used in the build process
-combiner_bin := bundle exec ./bin/combine_publisher_schema
```

Comment:
Should this PR delete `bin/combine_publisher_schema`? The only references to it I can find now are in its spec and the README.